src/http.zig

```diff
@@ -1556,7 +1554,9 @@ pub fn isKeepAlivePossible(this: *HTTPClient) bool {
         if (this.http_proxy != null and this.url.isHTTPS()) {
             return false;
         }
-        return !this.disable_keepalive;
+
+        //check state
```

Comment:
what is this comment trying to explain?

Reply:
It's pretty unnecessary, but saying that we are checking `this.state.allow_keepalive` (this comments was also present in the removed function I just keep it)